Doc updates (#3540)
input-output-hk/cardano-wallet: docs/user-guide/common-use-cases/delegation.md

-H "Content-Type: application/json"
```

> After constructing delegation quit transaction I should receive CBOR-encoded `transaction`, `fee` and `coin_selection` details in response.

Reviewer: ```suggestion
After constructing a quitting-from-pool transaction I should receive CBOR-encoded `transaction`, `fee` and `coin_selection` details in the corresponding response.
```

Reviewer 2: > "After constructing a transaction that undelegates from a stake pool, I should receive a CBOR-encoded `transaction`, `fee`, and `coin_selection` in the response."

I can now **sign** and **submit** such a transaction just like in [[how-to-make-a-transaction]].

> > :information_source: Note that all rewards will be withdrawn only if you add {"withdrawal":"self"} to the payload. You can do it in a single transaction though as above.

Reviewer: ```suggestion
> :information_source: Note that all rewards will be withdrawn only if you add {"withdrawal":"self"} to the payload. You can realize this within a single transaction as shown above.
```

Reviewer 2: > You can achieve this with a single transaction, as shown above.
